src/arduino/components/pressure.h

```diff
@@ -0,0 +1,9 @@
+class Pressure
```

Comment:
Should there be check pressure in range things for this class? is that a function we need here? or somewhere else